### ISSUE TITLE
Feature Request: Add Multi Group icon

### ISSUE THREAD

**Issue by hsayed21** [tjackenpacken/taskbar-groups#116]
Hi There,

Add an image icon that includes all the selected programs.
The program is great, but if an **image icon** was added to the group that includes all the **selected programs**, it would be more than wonderful.
And it appears for the user, which programs are in the group before it is pressed and discovered, so it is easy for it

**Comment by PikeNote** [comment]
For example, an icon can generate based on all the icons of the selected programs?

**Comment by hsayed21** [comment]
yes, that

**Comment by hsayed21** [comment]
like this
![alt text](https://cloud.addictivetips.com/wp-content/uploads/2011/03/Taskbar1.jpg)

**Comment by neoOpus** [comment]
please make this optional not by default!

**Comment by PikeNote** [comment]
Definitely, perhaps based on if there is any existing icons selected by the user.

**Comment by JohnLGalt** [comment]
I used BINS a lot back in the day, but it can't work with UWP apps at all.  So,  I'm exploring other options, and am trying TbG here to see if I can get something better.

I would like the option like in the screenshot above (which I'm betting is from BINS website), but would also like to make it optional as well - I found a nifty .PNG for browsers, for example, that I'd rather use than the 4 browsers' icons that I'm using for my group.

**Comment by PikeNote** [comment]
![image](https://user-images.githubusercontent.com/56088716/120911783-06e49f00-c658-11eb-8db5-65cf9259b8a6.png)
![image](https://user-images.githubusercontent.com/56088716/120911786-106e0700-c658-11eb-9a2a-5019cfeb61a3.png)

Feature implemented in #71 
Currently, it grabs the first four applications and turns them into an icon
Due to icon size restrictions, this is a bit small in the taskbar.
Will add a way to remove current group icon to regenerate it as user's please.

Any other suggestions, please add them here.

**Comment by Wolverine1977** [comment]
Is this part of  v0.2.0.0 ? or implemented to an upcoming release?

**Comment by hsayed21** [comment]
> ![image](https://user-images.githubusercontent.com/56088716/120911783-06e49f00-c658-11eb-8db5-65cf9259b8a6.png)
> ![image](https://user-images.githubusercontent.com/56088716/120911786-106e0700-c658-11eb-9a2a-5019cfeb61a3.png)
> 
> Feature implemented in #71
> Currently, it grabs the first four applications and turns them into an icon
> Due to icon size restrictions, this is a bit small in the taskbar.
> Will add a way to remove current group icon to regenerate it as user's please.
> 
> Any other suggestions, please add them here.

### Awesome thank you ❤💙 @PikeNote 
#### I will wait to add a way to remove the current group icon to regenerate it as user's please.

**Comment by Wolverine1977** [comment]
How do you activate this feature? I just set up Taskbar Groups and I'm not getting or seeing it as an option. Thanks :)

**Comment by hsayed21** [comment]
I think it will be implemented in an upcoming release

**Comment by Wolverine1977** [comment]
I loved Bins, this is close and showing a lot of promise. Is there a beta I can sign up for, I love testing new features!

**Comment by PikeNote** [comment]
Wish I had the knowledge or experience to implement half the magic of what Bins does. But I may consider releasing just beta builds of latest release somewhere.

**Comment by JohnLGalt** [comment]
You can do Betas right here in GitHub, I follow plenty of devs who do beta and final releases simultaneously.

As for Bins - yeah, it was a treat to use.  I worked with the dev for a while ascertaining it's worthiness in Windows 10, but after a while it started having too many issues.

**Comment by PikeNote** [comment]
I would release beta builds, but I don't have direct repo access or release access in this GitHub. I can't release beta builds alone. I do believe the repo owner would want to review changes, but they are a bit busy at the moment.

**Comment by JohnLGalt** [comment]
Whoops - I didn't even notice that you were not the repo owner - my bad.

Maybe a pull request and release Betas in a new repo?

**Comment by PikeNote** [comment]
https://github.com/PikeNote/taskbar-groups-pike-beta/releases/tag/v0.2.1.0

New "beta" release with the changes in pull #71. It is all 1 exe now. Please read the notice I put in there before you install for all the disclaimers and instructions/heads up.
